Lesson16/gozde/script.js

```diff
@@ -0,0 +1,263 @@
+// 1. Write a function that takes in a number and checks if it's even or odd using if else statement.
+
+function isEvenOrOdd(number) {
+	const reminder = number % 2;
+	// === - strict equality
+	if (reminder === 0) {
+		return 'Even';
+	} else {
+		return 'Odd';
+	}
+}
+
+console.log('Exercise 1', isEvenOrOdd(9), isEvenOrOdd(2));
+// 2. Write a function that takes age as input. If the age is less than 18, print "Sorry, you are not old enough to vote". If the age is between 18 and 65 (inclusive), print "You are eligible to vote". If the age is greater than 65, print "You are a senior citizen".
+function canVote(age) {
+	if (age < 18) {
+		return 'Sorry, you are not old enough to vote';
+	} else if (18 <= age && age <= 65) {
+		return 'You are eligible to vote';
+	} else {
+		return 'You are a senior citizen';
+	}
+}
+
+console.log('Exercise 2', canVote(10), canVote(30), canVote(75));
+
+// 3. Write a function that asks the user to enter two numbers and checks if the first number is greater than, less than, or equal to the second number using if else statement.
+function compareNumbers() {
```

Comment:
As I mentioned in the lesson, don't use 'prompt' function as it does not work in the NodeJs environment in the future (no need to change this exercise though). Instead, write functions in a way that they take input as parameters (the same approach that we use in the lessons).